www/js/main.js

```diff
@@ -96,8 +96,29 @@ angular.module('emission.main', ['emission.main.recent', 'emission.main.diary',
     }
 })
 
-.controller('ControlCtrl', function($scope, $window, $ionicScrollDelegate, $state, $ionicPopup, $ionicActionSheet, $ionicPopover, $rootScope) {
+.controller('ControlCtrl', function($scope, $window, $ionicScrollDelegate, $state, $ionicPopup, $ionicActionSheet, $ionicPopover, $rootScope, ControlHelper) {
+    $scope.emailLog = ControlHelper.emailLog;
     $scope.dark_theme = $rootScope.dark_theme;
+
+    
+    $scope.toggleLowAccuracy = function() {
+        if ($scope.settings.collect.config == null) {
+            return false; // config not loaded when loading ui, set default as false
```

Comment:
uh I mean the toggle function (toggleLowAccurafcy), returns bool, which is used in ng-check, to set toggle to the correct state. Another function (willUseLowAccuracy) is actually changing the state, as u mentioned. I mean the bool function is called multiple times, by someone I don't know. Maybe it's the scope.watch that ionic used to handle ng stuff?